### PR TITLE
Open see more add-ons in new tab

### PR DESCRIPTION
**Pull request by mstriemer**
Without this the page won't load since it isn't on `discovery.addons.mozilla.org`.

### CODE REVIEW

File: src/disco/containers/DiscoPane.js [hunk 1]
```diff
@@ -72,7 +72,9 @@ export class DiscoPane extends React.Component {
         </header>
         {results.map((item, i) => <Addon {...camelCaseProps(item)} key={i} />)}
         <div className="amo-link">
-          <a href="https://addons.mozilla.org/">{i18n.gettext('See more add-ons!')}</a>
+          <a href="https://addons.mozilla.org/" target="_blank">
```
Comment (muffinresearch) [inline src/disco/containers/DiscoPane.js]: Should add `rel="norefferer"` and `rel="noopener"`

Comment (muffinresearch) [inline src/disco/containers/DiscoPane.js]: Here's a demo since Matias' page doesn't have one: http://staticfil.es/moz/opener.html

**Comment by muffinresearch** [comment]
Nice catch, I wondered about this and then totally forgot to test it under the iframe.

r+wc

File: src/disco/containers/DiscoPane.js (hunk 1 above)
Comment (muffinresearch) [inline src/disco/containers/DiscoPane.js]: I suppose this is being very paranoid since we own the destination but still. 

*/me puts on tinfoil hat*

Comment (mstriemer) [inline src/disco/containers/DiscoPane.js]: I was thinking we might want to log that they came from the disco pane. Maybe that logging is better off in here though?

Comment (muffinresearch) [inline src/disco/containers/DiscoPane.js]: Yeah that would get implicitly logged via UA I would think. E.g. should show up as an exit page.

Comment (mstriemer) [inline src/disco/containers/DiscoPane.js]: Is that a good thing then or should I still add the `noreferrer`?

**Pull request merged**